src/utils/foldingBuffer.ts

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
should be `src/utils/foldingBuffer.ts` -> `src/utils/FoldingBuffer.ts`? or then opposite for `src/utils/FileBuffer.ts`?
referring to the capitalizing file name of the class

Reply:
FileBuffer should be renamed to fileBuffer will do this once I merge all the other PRs to avoid conflicts